components/docker-auth/cmd/authn/authentication.go

```diff
@@ -76,21 +94,31 @@ func main() {
 
 	if len(credentials) != 2 {
 		glog.Error("received more than two parameters")
-		glogFlushAndExit(extension.ErrorExitCode)
+		extension.GlogFlushAndExit(extension.ErrorExitCode)
 	}
 	uName := credentials[0]
 	token := credentials[1]
+	conf := extension.LoadConfig()
+	if isJWT(conf) {
+		validateJWT(token, uName)
+	} else {
+		if validateAccessToken(token, uName, conf) {
+			glog.Error(0)
```

Comment:
Shall we make this log line more meaningful? This does not provide any context and is not very useful.

Reply:
This is the place where we tell that user is successfully authenticated via docker auth. I have made the code more readable.